MAINTAINERS

```diff
@@ -0,0 +1,6 @@
+# Maintainers List
+(The list is sorted alphabetically)
+
+Edward Haas <edwardh@redhat.com>
```

Comment:
IMHO it makes sense to also add the GitHub username and the IRC handle (for Freenode) here.

Reply:
If we assume that there is someone who will take a look at the file and want to engage with us, it would be useful for them to know who is who on IRC or here in GitHub so they can easily reach out. The format could be:

``` Name <email> GitHub: @Github_handle IRC: IRC_handle ```

Reply:
I do not feel this is necessary. My GitHub user may be found through the member list there or by looking-up my email mentioned in this list.
Same goes for the IRC, and I do not expect anyone to contact me personally in 99% of the cases on IRC.

Lets keep it simple and trust users that reach the repo that they know how to communicate with the community.